src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -97,3 +103,42 @@ case class BreakdownResult(
     delta: Double)
 
 case class Failure(className: String, message: String)
+
+// KEEP ARGUMENTS SORTED BY NAME.
+// It simplifies lookup when checking if a parameter is here already.
+case class MLParams(
+    // *** Common to all algorithms ***
+    randomSeed: Option[Int] = Some(0),
```

Comment:
Set to something other than 0, or check that this is OK.  I recall running into some RNG in Spark which always generated 0 whenever given a seed of 0.